package.json

```diff
@@ -11,21 +11,21 @@
   },
   "engines": {
     "node": "16.x",
-    "pnpm": "*"
+    "pnpm": "7.x"
   },
   "scripts": {
-    "start": "pnpm start --filter demo",
-    "start:storybook": "pnpm start --filter storybook",
-    "build": "pnpm build --filter demo",
-    "build:storybook": "pnpm build --filter storybook",
-    "serve": "pnpm serve --filter demo",
-    "serve:storybook": "pnpm serve --filter storybook",
-    "packages": "pnpm build --filter {packages}",
-    "lint": "run-p lint:all:*",
-    "lint:all:eslint": "pnpm lint:eslint -r --parallel",
-    "lint:all:tsc": "pnpm lint:tsc -r --parallel",
-    "lint:eslint": "eslint \"**/*.{js,cjs,ts,tsx}\" --max-warnings=0",
-    "lint:tsc": "tsc --project tsconfig.lint.json  --noEmit && tsc --project cypress/tsconfig.json",
+    "start": "pnpm --filter demo start",
+    "start:storybook": "pnpm --filter storybook start",
+    "build": "pnpm --filter demo build",
+    "build:storybook": "pnpm --filter storybook build",
+    "serve": "pnpm --filter demo serve",
+    "serve:storybook": "pnpm --filter storybook serve",
+    "packages": "pnpm --filter {packages/*} build",
+    "lint": "run-p lint:**",
+    "lint:eslint": "pnpm -r --parallel lint:eslint",
+    "lint:tsc": "pnpm -r --parallel lint:tsc",
+    "lint:cypress:tsc": "tsc --project cypress/tsconfig.json",
+    "lint:root:eslint": "eslint \"**/*.{js,cjs,ts,tsx}\" --max-warnings=0",
```

Comment:
`lint:all:<eslint|tsc>` (`pnpm -r --parallel lint:<eslint|tsc>`) no longer runs the root `lint:<eslint|tsc>` script, which means that I was able to rename some of the root scripts:

- `lint:eslint` becomes `lint:root:eslint`
- `lint:tsc` becomes `lint:cypress:tsc`
- `lint:all:eslint` becomes `lint:eslint` 
- `lint:all:tsc` becomes `lint:tsc`

A couple of notes:

- The `lint` script now runs all the `lint:` scripts, including `lint:root:eslint` and `lint:cypress:tsc`
- I realised that running `tsc` on the whole workspace made no sense since we don't have `ts` files outside of the projects' `src` folders and `cypress` folder, which are all already covered by a `tsconfing` and `lint:tsc` script. We were actually running `tsc` twice on every file... Anyway, so that's why `lint:cypress:tsc` is called that and not `lint:root:tsc`, and that's why I removed the root `tsconfig.lint.ts`.

Reply:
> I realised that running tsc on the whole workspace made no sense since we don't have ts files outside of the projects' src folders and cypress folder, which are all already covered by a tsconfing and lint:tsc script.

Looking back at https://github.com/silx-kit/h5web/pull/911, I think it was to ensure also that the storybook was linted ? Is it still the case ?

Reply:
I don't think adding linting to the root was related to the Storybook, as the `storybook` project does have its own `lint:tsc` and `lint:eslint` scripts.

Reply:
Yes but these weren't called for some reasons.

It appears to be fine with the new setup though.